pages/components/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import styles from "@/styles/Home.module.scss";
+
+const Header = () => {
+  return (
+    <header className={styles.header}>
+      <div className={styles.header_wrap}>
+        <h1>
+          <Link href="/">
+            <Image src="/logo.png" alt="로고" width="150" height="51" />
+          </Link>
+        </h1>
+        <ul className={styles.header_menu}>
+          <li>
+            <Link href="/">자유게시판</Link>
+          </li>
+          <li>
+            <Link href="/additem">중고마켓</Link>
+          </li>
+        </ul>
```

Comment:
```suggestion
       <ul className={styles.header_menu}>
          {상수.map((item)=> <li>
            <Link href="/additem">중고마켓</Link>
          </li>)}
        </ul>

```
위처럼 하는것도 좋은 방법이에요. 이렇게 작성하면 상수에 값 하나만 추가해줘도 화면에 바로 렌더링이 될 수 있어요.